aries-site/src/pages/components/card.mdx

```diff
@@ -51,9 +59,64 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- When presenting high-level, summarized pieces of information.
+- When presenting an item or collection of items that can best be consumed visually.
+
 ## When not to use
```

Comment:
To me, they seem like peers not a parent/child relationship.

Reply:
open to whatever direction. My thinking was you really wouldn't present "when not to use" without the existence of "when to use," however "when to use" could exist on its own. Soft opinion.

Also, if we pair the guidance with the dos/donts, the heading might be extraneous.

Reply:
I'm going to leave as an h2. Visually, the h3 felt awkward on the page and since it doesn't significantly alter the semantic understanding of the page, I think leaving as an h2 is okay for now.